torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9344,6 +9360,91 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     OpInfo('scatter',
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_scatter,),
+    OpInfo('Tensor.bfloat16',
+           dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
```

Comment:
For ops that are method only, like contiguous, see here:

https://github.com/pytorch/pytorch/blob/6d4d636d66774eb803cffedd267849a8b8a3b536/torch/testing/_internal/common_methods_invocations.py#L6591

That's probably most convenient pattern to follow here, too?